Bring AC's rainbow text back

IanWen103026 opened the pull request:
- Rewrited rainbow color transition

ckefgisc28th committed:
Bring AC's rainbow text back

ckeisc43rd-cjtsai commented on app/assets/stylesheets/custom_styles.css:
```diff
@@ -17,7 +17,7 @@
 
 .btn-success {
-  background-color: rgb(88,185,158);
+  background-color:  rgb(88,185,158);
```
huh
my bad
maybe consider deleting it

ckeisc43rd-cjtsai commented on app/views/submissions/index.html.erb:
```diff
@@ -109,7 +109,7 @@ $("button:not([type='submit'])").attr("type", "button");
         <td id="time-<%= submission.id %>"><%= submission.total_time.to_i %></td>
         <td id="memory-<%= submission.id %>"><%= submission.total_memory.to_i %></td>
-        <td id="verdict-<%= submission.id %>" class="<%= verdict_class_map[submission.result] %>"><%= submission.result %></td>
+        <td id="verdict-<%= submission.id %>" class="<%= verdict_class_map[submission.result] %>" data-splitting><%= submission.result %></td>
```
whats this?

IanWen103026 replied:
For rainbow-text effect, split the texts

IanWen103026 committed:
Remove " "

IanWen103026 committed:
Fixed banner cache

- Fixed banner cache issue
- Move "Back to problem" button to the top
- Link contest's logo button to root_path

IanWen103026 committed:
Higher resolution logo

- Updated bannernew.png

IanWen103026 committed:
Fix the width of banner_new.png

IanWen103026 committed:
Problem Page Update

- Reformatting

IanWen103026 committed:
Update show.html.erb

- Reformatting

IanWen103026 committed:
Fixed padding

ckefgisc28th committed:
Remove "!important" from td, Fix wrong color in user page
